pkg/backend/httpstate/client/client.go

```diff
@@ -1714,8 +1714,15 @@ func (pc *Client) DownloadTemplate(ctx context.Context, downloadURL string) (io.
 	}
 
 	var bytes io.ReadCloser
-	header := make(http.Header, 1)
-	header.Add("Accept", "application/x-tar")
+	var header http.Header
+
+	// Only add Accept header if this is not a presigned URL to avoid signature validation failures
+	// See: https://docs.aws.amazon.com/prescriptive-guidance/latest/presigned-url-best-practices/identifying-requests.html
+	if !strings.Contains(downloadURL, "X-Amz-Expires=") {
+		header = make(http.Header, 1)
+		header.Add("Accept", "application/x-tar")
+	}
+
```

Comment:
I tried writing tests for this but it actually results in a fairly ugly refactor because this method instantiates its own new client. The suite could proxy requests to avoid hitting actual servers, but I believe neither option is great at the moment.